src/containers/ViewerPanel/selectors.test.ts

```diff
@@ -1,12 +1,12 @@
-import { initialState } from "../../../state/index";
```

Comment:
I took this test file out of the `test` folder because it was the only test file in there. Kinda irrelevant to this PR, sorry